pyproject.toml

```diff
@@ -57,5 +57,6 @@ module = [
     "pytorch_lightning.trainer.trainer",
     "pytorch_lightning.tuner.batch_size_scaling",
     "pytorch_lightning.utilities.data",
+    "lightning_lite.utilities.data",
```

Comment:
It's weird that you had to add this, `lite` is not checked yet (see L29)

Reply:
Yes it triggered, and added it because once someone fixes pytorch_lightning.utilities.data, we have the same changes for this file too.